app/Models/Post.php

```diff
@@ -53,7 +53,22 @@ public function signup()
      */
     public function reactions()
     {
-        return $this->hasMany(Reaction::class);
+        // return $this->morphToMany(Reaction::class, 'postable');
+        return $this->morphMany(Reaction::class, 'postable');
+    }
```

Comment:
Do the arguments for the failing `getAttribute` method call appear in the stack trace for that exception? If so, that might also give us some good ideas.